certformat_test.go

```diff
@@ -128,7 +128,7 @@ func mustParseCertificate(t *testing.T, filename string) *x509.Certificate {
 		t.Fatalf("failed to read %s: %v", filename, err)
 	}
 	block, rest := pem.Decode(pemData)
-	if block == nil || len(rest) > 0 {
+	if block == nil || len(rest) > 0 || block.Bytes == nil {
```

Comment:
The last condition is not really possible, but `len(block.Bytes) == 0` is, the following per file will return that:
```
-----BEGIN CERTIFICATE-----
-----END CERTIFICATE-----
```